server/routes/animalReport.js

```diff
@@ -21,4 +22,8 @@ router
     .put(authenticate, upload.single('image'), updateAnimalReport)
     .delete(authenticate, deleteAnimalReport)
 
+// router
```

Comment:
If these are no longer needed, please delete them as well to keep things clean and organized.

Reply:
Regarding the commented out lines of code